pkg/envtest/client/client.go

```diff
@@ -0,0 +1,60 @@
+package client
+
+import (
+	"fmt"
+	"net/url"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	kcapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+const (
+	envtestName = "envtest"
+)
+
+// KubeConfigFromREST reverse-engineers a kubeconfig file from a rest.Config.
+// The options are tailored towards the rest.Configs we generate, so they're
+// not broadly applicable.
+//
+// This is not intended to be exposed outside envtest for the above reasons.
```

Comment:
Question: The idea is to use this function only to convert envtest.Environment.Config to a kubeconfig, correct?

Could we instead just write a envtest.Environment.KubeConfig field when we set envtest.Environment.Config?

Reply:
Yes

Reply:
Yes

Reply:
Then let's do that. Then folks can simply consume env.KubeConfig and we don't have to expose a func to rest.Config to kubeconfig

Reply:
Done.